FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -1462,6 +1462,17 @@ - (void)useAppLanguage {
   });
 }
 
+- (void)useEmulatorWithHost:(NSString *)host port:(NSInteger)port {
+  NSAssert(host.length > 0, @"Cannot connect to nil or empty host");
+  dispatch_sync(FIRAuthGlobalWorkQueue(), ^{
+    self->_requestConfiguration.emulatorURL =
+        [NSString stringWithFormat:@"%@:%ld", host, (long)port];
```

Comment:
This won't work for IPv6 addresses as host. `::1` + `8080` should be `[::1]:8080` with IP in brackets.

Reply:
Done.